tests/features/editorial_workflow.feature

```diff
@@ -7,71 +7,66 @@ Feature: Corporate editorial workflow
     Given I am logged in as a user with the "Author" role
     When I visit "the demo content creation page"
     Then I should have the following options for the "Save as" select:
-      | Draft        |
-      | Needs Review |
+      | Draft |
     When I fill in "Title" with "Workflow demo"
     And I press "Save"
     And I visit "the content administration page"
     # The content is created and it is not published.
     Then I should see the text "not published" in the "Workflow demo" row
-    When I click "Edit"
```

Comment:
With this change we don't check anymore that an author can edit a node that is in draft. Let's add this back and also check that the only state available in the edit form when a content is in draft is "draft".

Reply:
We do.

Reply:
Where? :)